packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -193,6 +194,7 @@ describe('Table', () => {
             ...simpleTableData,
             {
                 stringData: 'a new string',
+                otherData: 'an even newer string',
```

Comment:
Maybe I missed it within this PR, but why do you need to add `otherData` to these records? I don't see it used.

Reply:
Removed. Not sure why I added that...must have been an initial idea for testing I abandoned (probably for the spec-driven tests).